features/nationaliteit.feature

```diff
@@ -35,38 +35,38 @@ Functionaliteit: Bepalen van de actuele nationaliteit van een persoon
       | datumIngangGeldigheid.datum       | 1975-07-07                                       |
       | datumIngangGeldigheid.langFormaat | 7 juli 1975                                      |
 
-      Scenario: persoon heeft meerdere nationaliteiten
+    Scenario: persoon heeft meerdere nationaliteiten
       Gegeven de persoon met burgerservicenummer '000000140' heeft een 'nationaliteit' met de volgende gegevens
       | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
-      | 0001                  | 001                   | 19750707                        |
+      | 0263                  | 301                   | 19620107                        |
```

Comment:
Wat is de reden van deze wijziging?

Reply:
het is niet mogelijk om 2 nationaliteiten in de BRP te hebben als 1 daarvan de Nederlandse nationaliteit is.